img/g/glovoapp.com.svg

```diff
@@ -0,0 +1 @@
v-.0336c0-3.44884 2.63736-6.36773 6.35661-6.36773 1.84838 0 3.07684.43921 4.2037 1.25095a1.0948458 1.0948458 0 0 1 .43947.87895 1.0702396 1.0702396 0 0 1-1.0705 1.07077c-.28178 0-.49583-.12409-.6649-.24792-.78899-.57494-1.65682-.96916-2.98662-.96916-2.27674 0-4.00103 1.99495-4.00103 4.35027v.0339c0 2.53577 1.66794 4.40664 4.19259 4.40664 1.16073 0 2.22012-.37201 2.97524-.93557v-2.31034h-2.43416c-.52997 0-.96944-.40561-.96944-.93557 0-.52969.43947-.94668.96917-.94668h3.43747c.61992 0 1.09326.47361 1.09326 1.09326v3.29089c0 .61992-.24792 1.05939-.75513 1.37504-1.05939.71014-2.53576 1.34144-4.38414 1.34144-3.86556-.0116-6.40159-2.76173-6.40159-6.34577m13.46808-5.75866a1.0511896 1.0511896 0 0 1 1.05939-1.05939c.59743 0 1.07077.47334 1.07077 1.05939v10.90957c0 .59716-.4736 1.05939-1.07077 1.05939a1.0511896 1.0511896 0 0 1-1.05939-1.05939zm3.78672 7.29165v-.0336c0-2.65985 2.10767-4.85775 4.95908-4.85775 2.85142 0 4.94771 2.16429 4.94771 4.82389v.0339c0 2.6371-2.10767 4.84611-4.98157 4.84611-2.82893-.0111-4.92522-2.17514-4.92522-4.81251m7.77663 0v-.0336c0-1.63433-1.18348-2.99799-2.85141-2.99799-1.71318 0-2.79506 1.34117-2.79506 2.95275v.0339c0 1.62295 1.18348 2.9755 2.82892 2.9755 1.72429.0111 2.81755-1.33006 2.81755-2.93053m7.44961 4.76753h-.10134c-.58605 0-.96943-.372-1.21708-.93556l-3.09959-7.08899c-.0564-.15769-.12383-.33813-.12383-.52969 0-.5297.47334-1.02553 1.05939-1.02553.58606 0 .87895.33814 1.05939.77761l2.38919 6.1648 2.43443-6.19866c.15769-.37174.43948-.74375 1.00304-.74375.58605 0 1.0369.43948 1.0369 1.02553 0 .19156-.0677.40587-.12409.5072l-3.09933 7.11148c-.24791.55218-.63103.93556-1.21708.93556m5.21811-4.76753v-.0336c0-2.65985 2.10767-4.85775 4.95909-4.85775 2.85115 0 4.9477 2.16429 4.9477 4.82389v.0339c0 2.6371-2.10767 4.84611-4.98157 4.84611-2.8403-.0111-4.92522-2.17514-4.92522-4.81251m7.77663 0v-.0336c0-1.63433-1.18348-2.99799-2.85141-2.99799-1.71318 0-2.79506 1.34117-2.79506 2.95275v.0339c0 1.62295 1.18322 2.9755 2.84004 2.9755 1.71318.0111 2.80643-1.33006 2.80643-2.93053" style="fill:#00a082;fill-rule:nonzero;stroke-width:.26458332" transform="translate(-100.56741698 -308.07330786)"/><path d="M151.99897 308.33816c-2.31008 0-4.18094 1.87086-4.18094 4.18121 0 .87921.2704 1.72429.78872 2.43443l.11271.15769 2.17514 3.0771s.27067.43921.84534.43921h.5297c.57494 0 .84534-.43921.84534-.43921l2.17488-3.0771.11298-.15769c.51858-.72126.78898-1.55522.78898-2.43417-.0114-2.31087-1.88224-4.18174-4.19285-4.18174m1.72429 5.42105-.11271.15769-1.60047 2.26536-1.61131-2.26536-.11298-.16907c-.25929-.37174-.4056-.78899-.4056-1.23957a2.1164021 2.1164021 0 0 1 2.11878-2.11878 2.1164021 2.1164021 0 0 1 2.11879 2.11878 2.1677312 2.1677312 0 0 1-.3945 1.25095m-2.89639 6.75111c0-.63103.49609-1.14962 1.16099-1.14962.67627 0 1.16099.50721 1.16099 1.13824v.0114c0 .61992-.49609 1.13824-1.1721 1.13824-.66517-.0111-1.14962-.51859-1.14962-1.13824" style="fill:#ffcc1b;fill-rule:nonzero;stroke-width:.26458332" transform="translate(-100.56741698 -308.07330786)"/></svg>
```

Comment:
We prefer to use the icon rather than the logo, as the icon normally looks better in small resolutions.
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18"><circle cx="9" cy="9" r="9" fill="#ffcc1b"/><path fill="#00a082" d="M9 2.6a4.02 4.02 0 0 0-3.27 6.36l.12.16 2.1 2.96s.26.41.8.41h.51c.56 0 .82-.41.82-.41l2.09-2.96.12-.16A4.04 4.04 0 0 0 9 2.6m1.66 5.23-.12.14L9 10.16 7.46 7.97l-.11-.16a2.02 2.02 0 0 1-.39-1.2A2.04 2.04 0 0 1 9 4.57a2.04 2.04 0 0 1 2.04 2.03 2.09 2.09 0 0 1-.38 1.22m-2.8 6.5c0-.61.49-1.11 1.13-1.11.66 0 1.12.48 1.12 1.1 0 .6-.48 1.1-1.13 1.1a1.1 1.1 0 0 1-1.1-1.1"/></svg>
```